src/queue.c

```diff
@@ -405,6 +405,62 @@ static int __io_uring_submit(struct io_uring *ring, unsigned submitted,
 	unsigned flags;
 	int ret;
 
+#if defined(__SANITIZE_ADDRESS__)
```

Comment:
I think that should go into a helper that is just stubbed out for !defined(__SANITIZE_ADDRESS__)